### ISSUE TITLE
Some Recommendations

### ISSUE THREAD

**Issue by e-dong** [1391819/MA-seek#1]
The documentation on this is great! I appreciate putting the work into this. I am looking to reproduce the experiments you did. It would be great if you included what versions of python dependencies you are using.

I believe these are the third party modules you are using based on looking at your source code:
* tensorflow
* keras
* numpy
* gym
* pygame

In the README, I would recommend adding a section on installation and usage

### Some Other Recommendations / Suggestions
Generally, `src` shouldn't be used to store source code: https://docs.python-guide.org/writing/structure/
You could have a package called `ma_seek` and a `setup.py` to allow one to `pip install -e ma_seek`.
`main.py`
This would allow you to write imports like this:
```
...
from ma_seek import PATH
from ma_seek.maenv import MASeek
from ma_seek.drqn import DRQN
from ma_seek.replay_buffer import ReplayBuffer
...
```

Instead of hardcoding configuration in `main.py`, you can take advantage of hydra for configuration management.
See https://hydra.cc/docs/intro/

**Comment by 1391819** [comment]
Hi @e-dong, 

thank you very much for your recommendations, greatly appreciated! Will definitely look into the things you mentioned as soon as I have some extra free time.

Unfortunately, right now I don't have access to the setup I actually implemented the project on (currently on vacation, will be back around the ~9th of January), so I'm not really able to fully look into what you requested. 

I'm 100% sure I did have a `requirements.txt` file in there, so not quite sure where it went, apologies. 

With that being said, I did this project around April 2022, and according to my brief research right now these could be the possible dependencies: 

- tensorflow==2.9.0
- gym==0.23.1
- keras==2.9.0
- numpy==1.23.5
- pygame==2.1.0

I will save this issue so I'll be able to get back to you with, most likely, the `requirements.txt` file as soon as I'm back home.
